code/build.gradle

```diff
@@ -24,7 +24,7 @@ ext {
 
     // test dependencies
     junitVersion = "1.1.3"
-    mockitoCoreVersion = "2.28.2"
+    mockitoCoreVersion = "4.5.1"
```

Comment:
we are renaming this to mockitoVersion in other edge extensions, can we make it consistent?